src/helpers/sender.js

```diff
@@ -8,23 +8,30 @@ dotenv.config();
 const conn = connection(configDB);
 
 exports.sendToFabric = async (username, method, props) => {
+  let response = null;
```

Comment:
it is better to use "undefined" in this case, because null has an unpredictable behaviour: https://twitter.com/brendaneich/status/1171079812179361792?lang=en

Reply:
fixed